imod/mf6/exchangebase.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum
+from typing import Dict, Tuple
+
+from imod.mf6.package import Package
+
+
+class ExchangeType(Enum):
+    GWFGWF = "GWF6-GWF6"
+    GWFGWT = "GWF6-GWT6"
+
+
+class ExchangeBase(Package):
```

Comment:
if it's a package, it should probably be included in the package sanity checks (test_package_sanity.py).

Reply:
Good point. Added it to the test